models/helpers.ts

```diff
@@ -190,6 +212,8 @@ export const statusColor: Record<
   NotSaved: 'gray',
   Submitted: 'green',
   Cancelled: 'red',
+  Return: 'orange',
+  ReturnIssued: 'gray',
```

Comment:
Should not be gray, this is to be used only for Draft entries

Reply:
Okay, would you happen to have any suggestions in mind?